src/main/java/LoginDelocalized/User.java

```diff
@@ -5,27 +5,31 @@ public class User {
     private String password;
 
     public User(String name, String password) {
-
+        this.name = name;
+        this.password = password;
     }
 
     public String getName() {
-
+        return name;
     }
 
     public void setName(String name) {
-
+        this.name = name;
     }
 
     public String getPassword() {
-
+        return password;
     }
 
     public void setPassword(String password) {
-
+        this.password = password;
     }
 
     @Override
     public String toString() {
-
+        return "User{" +
+                "name='" + name + '\'' +
+                ", password='" + password + '\'' +
```

Comment:
Should the security be addressed? Because the password handling is obviously problematic^^